webcrypto/tests/subtle_crypto_test.go

```diff
@@ -0,0 +1,197 @@
+// Part of the Web Platform Tests suite for the k6's WebCrypto API
+//go:build wpt
+
+package tests
```

Comment:
Maybe also explicitly name this package `wpt`?

Reply:
I have no strong opinion, but sounds good! 👍🏻